src/Symfony/Component/Messenger/Bridge/MongoDB/Tests/Stubs/mongodb.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+// stubs for mongodb/mongodb 1.17+
```

Comment:
I don't think we should define such stubs. Tests should use the actual library and be skipped otherwise.

Otherwise, we test that the component works against those stubs rather than against the actual library (hiding compat issues caused by new versions of the library)

Reply:
Yes, they are derived from [src/Symfony/Component/Lock/Tests/Store/stubs/mongodb.php](https://github.com/symfony/symfony/blob/7.4/src/Symfony/Component/Lock/Tests/Store/stubs/mongodb.php).

The issue is that we need both `ext-mongodb` and `mongodb/mongodb`. But by default the unit tests run without `ext-mongodb` so `mongodb/mongodb` blocks the installation. And integration tests run with `ext-mongodb` but then we need to re-add `mongodb/mongodb` dependency. 
This can be solved by adding `--ignore-platform-req=ext-mongodb` in the unit test job.